perf-tests/clusterloader2/pkg/measurement/util/object_store.go

```diff
@@ -74,27 +80,44 @@ func (s *ObjectStore) Stop() {
 // PodStore is a convenient wrapper around cache.Store.
 type PodStore struct {
 	*ObjectStore
+	listener int
 }
 
+var podStore *PodStore = nil
+var initPodStoreLock sync.Mutex
+
 // NewPodStore creates PodStore based on given object selector.
 func NewPodStore(c clientset.Interface, selector *ObjectSelector) (*PodStore, error) {
+	initPodStoreLock.Lock()
+	defer initPodStoreLock.Unlock()
+
+	if podStore != nil {
+		podStore.listener++
```

Comment:
we can delete the podstore at the end of the test to avoid using this listener counter.